packages/firestore/src/local/indexeddb_schema.ts

```diff
@@ -455,6 +419,20 @@ export class DbTargetGlobal {
   ) {}
 }
 
+export type DbInstanceKey = [string, string];
```

Comment:
comments?

Also, "Instance" feels too generic. Perhaps "ClientInstance" (and comments should explain the multi-tab scenario)

Reply:
No longer used.